edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/validation/AdvanceManifestFilingValidator.java

```diff
@@ -0,0 +1,24 @@
+package org.dcsa.edocumentation.transferobjects.validation;
+
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.dcsa.edocumentation.transferobjects.AdvanceManifestFilingTO;
+
+public class AdvanceManifestFilingValidator implements ConstraintValidator<AdvanceManifestFilingValidation, AdvanceManifestFilingTO> {
```

Comment:
Where is this Validator used?

Reply:
Its called when @Valid annotation is called in the controller (Spring context)